src/libxtp/threecenter_gwbse.cc

```diff
@@ -77,8 +110,24 @@ void TCMatrix_gwbse::Fill(const AOBasis& gwbasis, const AOBasis& dftbasis,
     }
```

Comment:
So this initialisation should be in the FillBlock routines, I do not mind the extra copy and you can rewrite it as 
`std::vector<Eigen::MatrixXd> block= std::vector<Eigen::MatrixXd>(_mtotal,Eigen::MatrixXd::Zero(_ntotal, shell.getNumFunc())`